src/App.tsx

```diff
@@ -1,61 +1,39 @@
 import './App.scss';
+import { AddTodoForm } from './components/AddTodoForm/AddTodoForm';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+
+function getUserByID(userId: number) {
+  return usersFromServer.find(user => user.id === userId);
+}
+
+const array = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserByID(todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState(array);
+
+  const addTodo = (title: string, userId: number) => {
+    const id = Math.max(...todos.map(todo => todo.id)) + 1;
```

Comment:
The calculation of `id` assumes that there is at least one todo in the list. If the list is empty, `Math.max` will return `-Infinity`, leading to an incorrect ID. Consider providing a default value to `Math.max` or handling the empty list case separately.